open-telemetry-addon/deploy/resources/cluster-management-addon.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ClusterManagementAddOn
+metadata:
+  annotations:
+    addon.open-cluster-management.io/lifecycle: addon-manager
+  name: otel
+spec:
+  addOnMeta:
+    displayName: otel addon
+  installStrategy:
+    type: Placements
+    placements:
```

Comment:
do we need to set this? I think it has been set on defaultConfigs.
cc @zhujian7